rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -202,6 +202,8 @@
      *             Include only jobs with a project name that starts with projectName (case in-sensitive)
      * @param userName
      *             Include only jobs with a user name that matches exactly with userName (case in-sensitive)
+     * @param parentId
+     *             Include only children jobs of the given parent job id. This parameter is ignored if childJobs == false
```

Comment:
This parameter is ignored if childJobs **is set to** false

Reply:
ok